docs/getting-started-guides/fedora/fedora_manual_config.md

```diff
@@ -70,44 +70,63 @@ KUBE_SERVICE_ADDRESSES="--portal_net=10.254.0.0/16"
 KUBE_API_ARGS=""
 ```
 
-* Edit /etc/kubernetes/controller-manager to appear as such:
+* Start the appropriate services on master:
+
+```
+for SERVICES in etcd kube-apiserver kube-controller-manager kube-scheduler; do
+	systemctl restart $SERVICES
+	systemctl enable $SERVICES
+	systemctl status $SERVICES
+done
 ```
-# The following values are used to configure the kubernetes controller-manager
 
-# defaults from config and apiserver should be adequate
+* Addition of nodes:
 
-# Comma separated list of minions
-KUBELET_ADDRESSES="--machines=fed-minion"
+* Create following node.json file on kubernetes master node:
 
-# Add you own!
-KUBE_CONTROLLER_MANAGER_ARGS=""
+```json
+{
+  "id": "fed-node",
+  "kind": "Minion",
+  "apiVersion": "v1beta1",
+    "labels": {
+    "name": "fed-node-label",
+  },
+}
```

Comment:
json is malformed
    trailing ',' after the label name
    trailing ','  and after the label '}'

Also whitespace seems a little nutty....